src/components/Settings/Settings.tsx

```diff
@@ -84,6 +99,7 @@ const Settings = () => {
                 minLength={1}
                 value={deviceNickname}
                 onChange={(e) => setDeviceNickname(e.target.value)}
+                disabled={selectActiveNode === null ? true : false}
```

Comment:
Nit: you don't need a ternary here, the condition already evaluates to true or false

Reply:
Added it because I got tired of seeing the red lines lol

Reply:
What red lines were you seeing? That should evaluate to a boolean expression

Reply:
Just realized I was evaluating the function instead of calling it; will make new commit to fix this

Reply:
Oh yeah, note that you shouldn't evaluate selectors inline. I would call it with the `useSelector` hook outside of the JSX then use that resulting value

Reply:
Looks like you could use the `activeNode` variable here

Reply:
That works, thanks!